openscap-report.spec

```diff
@@ -6,7 +6,8 @@ Version:        0.0.0
 Release:        0%{?dist}
 Summary:        A tool for generating human-readable reports from (SCAP) XCCDF and ARF results
 
-License:        LGPL-2.1
+# The entire source code is LGPL-2.1+ and GPL-2.0+ except schemas/ and assets/, which are Public Domain
+License:        (LGPLv2+ or GPLv2+) and Public Domain
```

Comment:
Is the possibility of using logical expression in this field documented anywhere? I wasn't able to find it by looking at https://rpm-packaging-guide.github.io/ and https://fedoraproject.org/wiki/Licensing:Main

Reply:
Here: https://docs.fedoraproject.org/en-US/packaging-guidelines/LicensingGuidelines/#_multiple_licensing_scenarios